PWM_NativePin.h

```diff
@@ -0,0 +1,41 @@
+#pragma once
+
+class PWM_NativePin {
```

Comment:
How to do PWM for physical pins. A class is a way to group functions.

Reply:
When you talk about "#pragma xxx" and "class xxx" I realize I have to go study "Explaining classes to Paul".... Learning is hard!